client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -72,6 +72,7 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
   // shuffle id -> (partitionId -> newest PartitionLocation)
   val latestPartitionLocation =
     JavaUtils.newConcurrentHashMap[Int, ConcurrentHashMap[Int, PartitionLocation]]()
+  private val shutdownWorkers: JSet[WorkerInfo] = ConcurrentHashMap.newKeySet()
```

Comment:
ApplicationHeartbeater is single-threaded, so I think we should not use ConcurrentHashMap. I think we should only use ConcurrentHashMap when multi-threading can happen.

Reply:
> ApplicationHeartbeater is single-threaded, so I think we should not use ConcurrentHashMap. I think we should only use ConcurrentHashMap when multi-threading can happen.

How about current?